doc/releases/release-notes-3.3.rst

```diff
@@ -2,13 +2,21 @@
 
 .. _zephyr_3.3:
 
-Zephyr 3.3.0 (Working Draft)
-############################
+Zephyr 3.3.0
+############
 
 We are pleased to announce the release of Zephyr version 3.3.0.
 
 Major enhancements with this release include:
 
+* Introduced :ref:`Fuel Gauge <fuel_gauge_api>` subsystem for battery level
+  monitoring.
+* Introduced :ref:`USB-C <usbc_api>` device stack with PD (power delivery)
+  support.
+* Introduced :ref:`DSP (digital signal processing) <zdsp_api>` subsystem with
+  CMSIS-DSP as the default backend.
+* Added Picolibc support for all architectures.
```

Comment:
It doesn't matter a whole bunch, but it might be worth mentioning that this support is limited to the Zephyr SDK toolchain (no XCC) (https://github.com/zephyrproject-rtos/zephyr/pull/54523/files)

Reply:
Done